web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/QuestionIsHandledValidator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.csintegration.integration;
+
+import org.springframework.stereotype.Component;
+import se.inera.intyg.webcert.persistence.model.Status;
+import se.inera.intyg.webcert.web.web.controller.api.dto.ArendeListItem;
+
+@Component
+public class QuestionIsHandledValidator {
```

Comment:
I don't understand how QuestionIsHandledValidator and QuestionStatusValidator works. Would be great if we could go through the implementation together. Also why do we need to have Reminders here in this class? No ArendeListItem is a reminder if we fetch them from CS.

Reply:
Refactored validators